src/main/java/com/onaio/steps/model/Household.java

```diff
@@ -45,7 +45,9 @@
     public static final String SERVER_STATUS = "Server_Status";
     public static final String UNIQUE_DEVICE_ID = "unique_device_id";
     public static final String ODK_FORM_ID = "odk_form_id";
-    public static final String TABLE_CREATE_QUERY = String.format("CREATE TABLE %s(%s INTEGER PRIMARY KEY, %s TEXT, %s TEXT, %s INTEGER, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT)", TABLE_NAME, ID, NAME, PHONE_NUMBER, SELECTED_MEMBER_ID,STATUS, CREATED_AT ,COMMENTS, UNIQUE_DEVICE_ID, SERVER_STATUS, ODK_FORM_ID);
+    public static final String ODK_JR_FORM_ID = "odk_jr_form_id";
+    public static final String ODK_JR_FORM_TITLE = "odk_jr_form_title";
+    public static final String TABLE_CREATE_QUERY = String.format("CREATE TABLE %s(%s INTEGER PRIMARY KEY, %s TEXT, %s TEXT, %s INTEGER, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT)", TABLE_NAME, ID, NAME, PHONE_NUMBER, SELECTED_MEMBER_ID,STATUS, CREATED_AT ,COMMENTS, UNIQUE_DEVICE_ID, SERVER_STATUS, ODK_FORM_ID, ODK_JR_FORM_ID, ODK_JR_FORM_TITLE);
```

Comment:
Is the app an upgrade to current app installations. If so, we need some migrations for this

Reply:
Yes, it should be an upgrade to existing installation. Let's add the necessary migrations.

Reply:
@ukanga when we will upgrade the database then we have to set the `form-id` and `form-title` in this case which `form-id` and `form-title` will be used to set all previous households?

Reply:
Let's use `STEPS_Instrument_V3_1` for both for now.

Reply:
Also, let's ensure we can change this on building the APK without having to change much of the code.

Reply:
Okay, cool

Reply:
now loading default `form-id` from `gradle.properties` file.

Reply:
@owais-vd Were the migrations for this added?

Reply:
Yes migration added in DatabaseHelper class